composer.json

```diff
@@ -56,9 +55,8 @@
         "pear/archive_tar": "^1.4.12",
         "pelago/emogrifier": "^2.1",
         "phpoffice/phpspreadsheet": "~1.5",
-        "prestashop/blockreassurance": "^5",
```

Comment:
Please create issues for blockreassurance & productcomments. And it's ok for me.

Reply:
Exists already: https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue++circuit-breaker

Reply:
OK for me.